packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1346,6 +1346,7 @@
   "If_you_are_sure_type_in_your_password": "If you are sure type in your password:",
   "If_you_are_sure_type_in_your_username": "If you are sure type in your username:",
   "If_you_dont_have_one_send_an_email_to_omni_rocketchat_to_get_yours": "If you don't have one send an email to [omni@rocket.chat](mailto:omni@rocket.chat) to get yours.",
+  "If_you_didnt_ask_for_reset_ignore_this_email" : "If you didn't ask for reset your password, don't worry, just ignore this email",
```

Comment:
> If you didn't ask for your password reset, don't worry, just ignore this email

would be more proper placement of reset.

> If you didn't ask for your password reset, you can ignore this email.

But to me this would be better :top:

Reply:
I still don't think we should say to ignore 😛 it _might_ indicate a tentative of account steal.